application.md

```diff
@@ -1,131 +1,108 @@
-students. 
-A number of the mentors have been GSoC mentors in 2013 and 2014.
-
-## What is your plan for dealing with disappearing students?
-
-Students are expected to be in regular contact with the mentors assigned and
-with the community via the project’s regular communication channels. 
-This include IRC, mail-list, or dev-Google+ hangouts. 
-The students will also be required to write blog posts on a regular basis on the
-OpenAstronomy site. 
-Additionally, the mentors and the student have to agree on a time to meet every
-week for at least one hour to discuss the evolution of the project. 
-The students are also expected to have their repositories on GitHub or BitBucket
-up to date, so the mentors and all community members can review the progress as
-it proceeds.
-
-If a student simply stops communicating, or otherwise does not show progress,
-the student will be explicitly warned that their project is in jeopardy via
-e-mail (if they have stopped communicating) or in the regular weekly meeting.
-If they still do not respond or return to making progress, the mentors and org
-admins will terminate the project and give the student a failing grade, unless
-the student has extenuating circumstances.
-Any code the student has developed up to that point will be present in the
-GitHub or BitBucket repositories, so that their project can be continued by
-project developers or a hypothetical future GSoC student.
-
-
-## What is your plan for dealing with disappearing mentors?
-
-All students will be assigned two mentors in case one of them becomes
-unavailable at any point. 
-We will also ensure that the students regularly interact with the main developer
-list and communicate with others on the project, so that in the unlikely case
-that both mentors become unavailable, other mentors and developers will be able
-to step in and help the student.
-
-In case either of the mentors have either a temporary or permanent conflict
-preventing them from completing their mentorship role, the sub-organisation’s
-administrator will be responsible for finding a new mentor. 
-The projects within the OpenAstronomy project are large enough that it will not
-be difficult to find another mentor that is familiar with the project.
-
-## What steps will you take to encourage students to interact with your project's community before and during the program?
+students.
+A number of the mentors have been GSoC mentors in all the editions since 2013.
+How will you keep mentors engaged with their students? Each sub-organisation has
+weekly teleconferences to discuss theIr development, in these calls they will
+dedicate a time for the GSoC mentors and students to update the progress and
+problems encountered.
+
+/is that engaging enough? A Party or chocolates will have more effect,
+something that will make others to want to mentor in the future/
+
+## How will you help your students stay on schedule to complete their projects?
```

Comment:
The question has changed tone, it's not asking anymore about how to deal with disappearing, but how to help to stay on schedule to complete.